src/entry.js

```diff
@@ -16,7 +16,7 @@
 
 'use strict';
 
-var commonGrpc = require('@google-cloud/common-grpc');
+var {Service} = require('@google-cloud/common-grpc');
```

Comment:
const?

Reply:
I filed #204 to come back around and update this everywhere 👍